base_geoengine/static/src/js/views/geoengine/geoengine_controller.js

```diff
@@ -16,7 +16,7 @@ odoo.define("base_geoengine.GeoengineController", function (require) {
     var core = require("web.core");
     var BasicController = require("web.BasicController");
     var pyUtils = require("web.py_utils");
-    var Sidebar = require("web.Sidebar");
+    var Sidebar = require("web.ActionMenus");
```

Comment:
Don't we need to change variable name as well appropriate to "ActionMenus"?

Reply:
Right! Fixed rn